lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -27,9 +27,12 @@
       - "A string expression of the same form that can be passed to the 'when' statement"
       - "Alternatively, a list of string expressions"
     required: true
-  msg:
+  fail_msg:
+    version_added: "2.7"
```

Comment:
you might also want to explain that 'msg' (now an alias) was the main way to use the parameter pre 2.7